README.md

```diff
@@ -126,13 +126,12 @@ from anomalib.data import MVTec
 from anomalib.models import Patchcore
 from anomalib.engine import Engine
 
-# Initialize the datamodule, model and engine
+# Initialize the datamodule and engine
 datamodule = MVTec()
-model = Patchcore()
-engine = Engine()
+engine = Engine(model=Patchcore())
```

Comment:
I think this change should not change the default behaviour, so this document could stay as is. If we want to mention this feature, this could be an additional example.